src/components/Calculator.js

```diff
@@ -1,50 +1,65 @@
+/* eslint-disable */
 import React, { Component } from 'react';
+import Btn from './Button';
+import calculate from '../logic/calculator';
 
 class CalculatorRender extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.Val = [
+      { btn: 'AC', style: 'btn grey' },
+      { btn: '+/-', style: 'btn grey' },
+      { btn: '%', style: 'btn grey' },
+      { btn: '/', style: 'btn orange' },
```

Comment:
- [ ] It should be good if you check very well the functionality of this operation sign `/` because we are not getting answers when you use it.
- [ ] When we are using it, we are getting an error when we need to calculate the answer  that has gotten with a new value